dask_ml/model_selection/methods.py

```diff
@@ -276,19 +276,135 @@ def fit_transform(
     return (est, fit_time), Xt
 
 
-def _score(est, X, y, scorer):
+def _get_fold_sample_weights(sample_weight, cv, n):
+    """Get the training and test sample weights for the ``n`` th fold using
+    ``cv``.
+
+    Parameters
+    ----------
+    sample_weight : array-like
+        sample weights.  Should contain all sample weights needed for
+        training and testing. May be None.
```

Comment:
Why is it sensical for `sample_weight` to be `None` in a function called `_get_fold_sample_weights`?